From 41e1c3c0c1912e02528fca73fa21cf21abd012fb Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan@agner.ch>
Date: Thu, 16 Aug 2018 16:38:26 +0000
Subject: [PATCH 2/2] etnaviv: use different error code

The return code LDR_NOHARDWARE has been dropped in X.Org Server
1.20. Use LDR_MODSPECIFIC instead.

Signed-off-by: Stefan Agner <stefan@agner.ch>
---
 etnaviv/etnadrm_module.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etnaviv/etnadrm_module.c b/etnaviv/etnadrm_module.c
index 4307ff7..5e97cee 100644
--- a/etnaviv/etnadrm_module.c
+++ b/etnaviv/etnadrm_module.c
@@ -27,7 +27,7 @@ static pointer etnadrm_setup(pointer module, pointer opts, int *errmaj,
 		return (pointer) 1;
 	}
 
-	*errmaj = LDR_NOHARDWARE;
+	*errmaj = LDR_MODSPECIFIC;
 	*errmin = 0;
 
 	return NULL;
-- 
2.18.0

